Gemfile.lock

```diff
@@ -955,21 +998,24 @@ GEM
     websocket-driver (0.6.5)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    xml-simple (1.1.8)
+    xml-simple (1.1.9)
+      rexml
     xpath (3.2.0)
       nokogiri (~> 1.8)
     xray-rails (0.3.2)
       rails (>= 3.1.0)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-19
+  x86_64-linux
```

Comment:
ran `bundle lock  --add-platform x86_64-linux` because the CI pipeline kept failing

![Screen Shot 2022-02-28 at 5 09 00 PM](https://user-images.githubusercontent.com/29032869/156074426-7ab4f854-f161-49fa-a3ff-999f47dd7295.png)